lib/page-object/page_populator.rb

```diff
@@ -73,10 +84,15 @@ def is_radiobuttongroup?(key)
       respond_to?("select_#{key}".to_sym) and respond_to?("#{key}_values")
     end
 
+    def is_select_list?(key)
+      respond_to?("#{key}_options".to_sym)
+    end
+
     def is_enabled?(key)
       return false if is_radiobuttongroup?(key)
-      return true if (self.send "#{key}_element").tag_name == "textarea"
+      return false if is_select_list?(key)
```

Comment:
I don't remember , this was 2 years ago......ohhh i know , you want to ignore text population if it's select list since it will fail , select will return true for select list .... i guess i should add changes in 'is_text?' rather then enable....